src/components/DayOfCode.tsx

```diff
@@ -5,19 +5,19 @@ import { HeaderSections } from './shared/PlaynetConstants';
 import './styles/Feedback.scss';
 
 function DayOfCode(): JSX.Element {
-  const link_to_checklist = 'http://links.uclaacm.com/dayofcodechecklist';
+  const link_to_checklist = 'http://links.uclaacm.com/dayofcodepassport';
   return (
     <Base section={HeaderSections.DAY_OF_CODE}>
       <div id={'feedback-container'}>
         <p>
           The code for this event is:
-          <h1><i>C O D</i></h1>
+          <h1><i>C O D I N G</i></h1>
           <button className='playnet-button'>
             <a href={link_to_checklist} target='_blank' rel='noreferrer'>Add it to your passport!</a>
           </button>
```

Comment:
out of scope of this pr, but could you help add this in? thanks!!
```suggestion
            <a href={link_to_checklist} className='playnet-button' id='feedback-link'  target='_blank' rel='noreferrer'>Add it to your passport!</a>
```